subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1528,6 +1537,13 @@ static uint8_t le_controller_cmd_put(uint8_t const * const cmd,
 									(void *)event_out_params);
 #endif
 
+#if defined(CONFIG_BT_CTLR_SUBRATING)
+	case SDC_HCI_OPCODE_CMD_LE_SET_DEFAULT_SUBRATE:
+		return sdc_hci_cmd_le_set_default_subrate((void *)cmd_params);
```

Comment:
Set default subrate can only be called by a central, so probably best to guard this with CONFIG_BT_CENTRAL, same https://github.com/nrfconnect/sdk-nrf/pull/15500/files#diff-f5bf9d2614b19c06ca472417c8004daeae8125af4a59c8ec6983180c9ddae3daR613